src/Admin/Sass/site.scss

```diff
@@ -49,3 +53,11 @@ h3 {
 .form-check-input {
     margin-top: .45rem;
 }
+
+a {
```

Comment:
Bootstrap 5 automatically adds an underline to hyperlinks. This change removes the underline to match the Bootstrap 4 style.